WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -52,23 +51,25 @@ public static int Main(string[] args)
 				AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
 				TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
 
-				if (args.Length != 0)
+				if (crashReporter.ProcessCliArgs(args) is { } exceptionToReport)
 				{
-					ProcessCliCommands(Global, args, crashReporter);
+					// Show the exception
+					Console.WriteLine($"TODO Implement crash reporting. {exceptionToReport}");
 				}
 				else
 				{
-					if (crashReporter.IsReport)
+					if (args.Length != 0)
 					{
-						Console.WriteLine("TODO Implement crash reporting.");
-						return 0;
+						ProcessCliCommands(Global, args);
```

Comment:
The function returns bool but it is not processed.

Reply:
Out of scope of this work but I would really like to be able to allow setting logger log level via CLI. It would be very user-friendly.

Reply:
ProcessCliCommands will be removed.